app/javascript/packs/new_service.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import {render} from 'react-dom'
+import NewServiceWrapper from 'NewService/index'
+
+document.addEventListener('DOMContentLoaded', () => {
+  const newServiceWrapper = document.getElementById('new_service_wrapper')
+  const newServiceData = JSON.parse(newServiceWrapper.dataset.newServiceData)
```

Comment:
you could use our json_utils here. Maybe decide what to do when the config is wrong...